elementary/clients/dbt/dbt_runner.py

```diff
@@ -18,11 +19,13 @@ def __init__(
         profiles_dir: str,
         target: Optional[str] = None,
         raise_on_failure: bool = True,
+        dbt_quoting: bool = None,
```

Comment:
See my comment on `_get_command_env`
I think it will be better to pass dbt_env_vars/env_vars